lib/Backend/IRType.h

```diff
@@ -39,3 +37,14 @@ extern IRType IRType_EnsureUnsigned(IRType type);
 #if DBG_DUMP || defined(ENABLE_IR_VIEWER)
 extern void IRType_Dump(IRType type);
 #endif
+
+#ifdef _M_AMD64
+    #include "../../Backend/amd64/machvalues.h"
```

Comment:
Did you try non-relative paths here? Both on *nix and Windows we define a good range of search path.

Reply:
It seems as if the modules outside of lib/Backend don't have the architecture specific directory in their include path, so the specific path for the architecture has to be named explicitly.

I was therefore imitating the style used in lib/Backend/BackendOpCodeList.h, which does a similar arrangement of #includes for the appropriate platform for including MdOpCodes.h

Reply:
`amd64/machvalues.h` instead of `../../Backend/amd64/machvalues.h` ? (same applies to others)